orbit/cmd/desktop/desktop.go

```diff
@@ -83,7 +83,7 @@ func main() {
 				defer close(done)
 
 				for {
-					_, err := client.ListDevicePolicies()
+					_, err := client.GetDesktopPayload()
```

Comment:
Won't this break Fleet Desktop as soon as we push new Orbit+Desktop? (If customer is using Fleet server < 4.21.0)

@chiiph @roperzh

Reply:
Hm... right, we need to do the capabilities dance here. Great catch Lucas, I totally missed this

Reply:
Great catch indeed, I can tackle this alongside the other capabilities related work

Reply:
As in, it might be better for @juan-fdz-hawa to go through the exercise of adding that part. But if it creates too much conflict with your code, ignore me.

Reply:
That's a great point 👍, and sounds good to mr

Reply:
Is there any issue or code I can reference to figure out what needs to be done?

Reply:
not yet, I'll ping you to review the code once we have a PR ready 👍

Reply:
Thanks

Reply:
here's a PR https://github.com/fleetdm/fleet/pull/7833